### DISCUSSION TITLE
de W7DGP problem with "blue board" testing

### DISCUSSION THREAD

**Discussion by W7DGP** [alanbjohnston/CubeSatSim#159] [Q&A]
I'm up to the point of testing the INA219s. I have blue boards (LOTS of them) and set up the test fixture J4. The results are not good. The first pack of 6 show 3 boards with 0V, 0 ma and the remaining 3, .6 to .9V and zero current. So, voltage is fine, but 0 mA doesn't meet the spec.
My question: is 0 mA ok for these blue boards? If not, why is a whole package 0 mA? 
I am using the command line to test: CubeSatSim/telem  Looking at the schematic, I see the I2C bus pulled up and so checked continuity from one end of the resistor to SDA and SCL around the board. There is continuity from the resistors to most SDA and SCL on the board but not everywhere. So, my first step to see if the blue board under test is connecting ot the bus. It has to be to get any reading, right? If it gets a voltage reading but not a current reading, the bus has to be ok but addressing may be off. That's software. Obviously, I can use some guidance with this problem.

All tests up to this point work well, Its fun to hear the various modulations and my call on 434.902 MHz. On the Pi, I'm using the AMSAT SD card. I did go in and set up WIFI to make it easier to access. 
 
Not as critical is the Foxtelem app. I have it working and use HRD with an IC-9700. I believe I have the sources and level are set ok but I get all zeros ("all balls") and don't see the .9v from the blue board, for example. The "satellite" selected is CubeSatSim-FSK. The modulation method from the CubeSat is FSK. On playback, I get a poor eye and no data. I do have a FunCube Dongle Pro+ that I can use instead of my 9700. That may be easier to set up and get going.

**Comment by alanbjohnston** [comment]
Hi! 0 mA is fine as long as the voltage isn’t 0 V. I have edited the Wiki as it gave the misleading impression that 0 mA is a bad board. 

The IC-9700, like many radios, filters out low frequencies when in FM mode (e.g. PL tones, etc) which also filters out the DUV/FSK telemetry.  But you can get it using the IF output. I couldn’t find the full instructions searching but I did find this: https://github.com/ac2cz/FoxTelem/issues/629

The FunCube will work very well as SDRs don’t filter the audio unless you tell it. For example, the SDR# application has a Filter Audio button. If you check it, the DUV signal also goes away. 

Good luck and 73!

Alan

**Comment by W7DGP** [comment]
Thanks Alan. I’ll continue to test boards then.  Good to know. Also, I’ll change over to the FunCube dongle and SDR#. I’ve had that working well in the past.
73, Larry

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by W7DGP** [comment]
With all this troubleshooting, I found a better method to mount the blue boards on the test fixture and find that the entire package of 6 boards are good. All show voltage. Time to install them!  With the exception of the solar panels, I believe I have the entire BOM at hand. The 3D printed frames were ordered with all in different colors and really fit together nicely. Next week the solar panels should arrive.
